src/objects.js

```diff
@@ -1,41 +1,66 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name, 
+    age: age
+  }
 };
 
 const getName = object => {
-  // your code here
+  return object.name
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property]
+  // use square bracket notation to access a property using its variable name 
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]){
+    return true 
+  }
+  return false
```

Comment:
This could be refactored with the ternary operator. OR

You could use the double NOT operator.
object[property] will return the property of the object. Let's use the first test case in your tests, it uses the object Fred and checking if Fred has the property age. The outcome has to be true.
object[property] will return 79.
79 is a number not a boolean what we are looking for here. We need to return a boolean based on a truthy value we have which is the number(79).
You can do this with an if statement using ternary operator as you did in your solution OR
you could return 
`````
return !!object[property]
``````
object[property] = 79
!object[property] = false (as you are saying NOT 79)
!!object[property] = true (as you are saying NOT NOT 79 = 79)